crates/ironrdp-client/src/config.rs

```diff
@@ -310,6 +319,44 @@ impl Config {
             }
         }
 
+        let mut gw: Option<GwConnectTarget> = None;
+
+        let gw_addr = if let Some(gw_addr) = args.gw_endpoint {
+            gw_addr
+        } else {
+            inquire::Text::new("Gateway address:")
+                .prompt()
+                .context("Address prompt")?
+        };
```

Comment:
issue: According to this logic, we’ll always be prompted for the gateway address unless we specify it in the CLI. I would prefer we preserve the current behavior when none of these arguments are specified, as I think not using an RD Gateway is the most frequent use case.

suggestion:
```suggestion
        let mut gw: Option<GwConnectTarget> = None;

        let gw_addr = args.gw_endpoint;
```